### PR TITLE
excel.paste command

### PR DESCRIPTION
**Pull request by MarekKosinski**

### CODE REVIEW

**Review by folbo — APPROVED** [review]

File: G1ANT.Addon.MSOffice/Commands/ExcelPasteCommand.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+*
+*/
+
+using System;
+using System.Linq;
+using G1ANT.Language;
+
+namespace G1ANT.Addon.MSOffice
+{
+    [Command(Name = "excel.paste", Tooltip = "This inserts clipboard content into the currently selected cell or range")]
```
Comment (Vegebutcher) [inline G1ANT.Addon.MSOffice/Commands/ExcelPasteCommand.cs]: This command

**Review by Vegebutcher — APPROVED** [review]

File: G1ANT.Addon.MSOffice/Commands/ExcelPasteCommand.cs (hunk 1 above)
Comment (MarekKosinski) [reply G1ANT.Addon.MSOffice/Commands/ExcelPasteCommand.cs]:
```suggestion
    [Command(Name = "excel.paste", Tooltip = "This command inserts clipboard content into the currently selected cell or range")]
```

**Review by pwitusik — APPROVED** [review]

File: G1ANT.Addon.MSOffice/Commands/ExcelPasteCommand.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+*
+*/
+
+using System;
+using System.Linq;
+using G1ANT.Language;
+
+namespace G1ANT.Addon.MSOffice
+{
+    [Command(Name = "excel.paste", Tooltip = "This command inserts clipboard content into the currently selected cell or range")]
```
Comment (lfronczyk) [inline G1ANT.Addon.MSOffice/Commands/ExcelPasteCommand.cs]: Remove "This command"

**Review by lfronczyk — APPROVED** [review]

File: G1ANT.Addon.MSOffice/Commands/ExcelPasteCommand.cs [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+        public void Execute(CommandArguments arguments)
+        {
+            try
+            {
+                ExcelManager.CurrentExcel.Paste();
+            }
+            catch (Exception ex)
+            {
+                throw new ApplicationException($"Error occured while trying to pasting. Message: {ex.Message}", ex);
```
Comment (Vegebutcher) [inline G1ANT.Addon.MSOffice/Commands/ExcelPasteCommand.cs]:
```suggestion
                throw new ApplicationException($"Error occurred while trying to paste. Message: {ex.Message}", ex);
```

**Pull request merged**